### ISSUE TITLE
Use Transifex for continuous translation please

### ISSUE THREAD

**Issue by p-bo** [powerpaul17/nc_money#4]
Hello,
would it be possible to follow instructions on:
https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#adding-translations
...and gain benefits from work of whole Nextcloud translators community for users of your app please?

For instance:

- much more translators will get involved, because without it, they will newer even know about possibility to translate this app - example of benefits for project known in this manner for community of Nextcloud translators (app with recently added to this workflow, based on same request as this one): https://github.com/H2CK/oidc/commit/937fc37d7797fecc9f4f49e12a154cd8987cf99c (se this and ongoing Nextcloud bot commits - translation of app is getting completed/improved in no time)
- also, as app is evolving, when some string(s) in app is/are changed/added, in case of continuous translation workflow translators are noticed intermediately and strings are translated before new version is released. In contrast with situation, that this change (in source strings) is only silently published in git repo and no one will (get) notice(d) till never version with untranslated strings get released (proactive vs retroactive).
- when one (of translators) will during using this app notice possibility to translate something in better manner, it is much efficient (especially for minor changes) to do this in translation tool than edit file directly in git. For example in case of one letter changes (misspelled words, etc.) has change directly in git such overhead (many steps to change just one letter), that these type of tweaks are rare due this (loss of quality for users)

To conclude: with continuous translation workflow, result is, that users of your app are getting more complete and more polished localization, so their user experience is much more pleasant.

**Comment by powerpaul17** [comment]
Hey, although I'm not a big fan of such services I can see that it would make the process of translation easier and the app better for a lot of people. It is not my top priority at the moment since I haven't even released version 1 yet. But I'll have a look at it..

**Comment by powerpaul17** [comment]
@rakekniven, thanks for your answer on Transifex 🙂 . Here's the issue.. 😉

**Comment by powerpaul17** [comment]
> Just seen that last commit has been done some years ago. Will you continue to develop your app?

Yeah, I'm actually developing right now, changes are currently on `complete_rework` branch :wink:

**Comment by powerpaul17** [comment]
> The bot will not pollute your repo. It writes in just one folder calles "l10n".

I meant that the bot is committing daily if only a word or sentence changed. I really would like to pull in all translation changes before preparing a new release

> This is not clear to me.
See https://github.com/nextcloud/activity/tree/master/l10n
There are two files for each language, one is json.

That was not clear to me, I thought I have to use `gettext` to extract everything into `.po` files or so.. Also my translation files live in `src/locales` since I only need them in the frontend..

**Comment by powerpaul17** [comment]
Ok, I thought about it and read the documentation again and I decided to use Nextcloud's translation system. I set up the `l10n` directory and included `@nextcloud/l10n`. Only thing left is to let the bot do its job, right? Do I also need to create a Transifex configuration as well?

**Comment by powerpaul17** [comment]
Wow great, thanks for the quick answer :slightly_smiling_face:

**Comment by powerpaul17** [comment]
Yes :slightly_smiling_face:

**Comment by powerpaul17** [comment]
Thanks, I'm already impressed by the quick translation updates.. 🤩
